inc/integrations/class-wordpress-seo.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * WP Irving integration for WordPress_SEO.
+ *
+ * @package WP_Irving;
+ */
+
+namespace WP_Irving;
+
+/**
+ * Class to add WordPress_SEO's query modifications.
+ */
+class WordPress_SEO {
```

Comment:
This looks good, the code in the constructor and the `get_webmaster_tools_codes()` method should be able to be taken from here and put into my `Yoast` class once #108 is merged in.